exit status: 1

stdout = """
def error: Type = {x: Int, y: Bool};
types of expressions:
12..36 = Type
16..19 = Type
24..28 = Type
33..35 = <missing>
"""

stderr = """
error: duplicate field `x` in record type
  ┌─ tests/elab/errors/record_type/duplicate_fields.pion:1:31
  │
1 │ def error = {x: Int, y: Bool, x: ()};
  │              -                ^ duplicate field
  │              │                 
  │              first field

aborting due to 1 previous error
"""
